railties/lib/rails/generators/rails/app/templates/package.json.tt

```diff
@@ -2,9 +2,10 @@
   "name": "<%= app_name %>",
   "private": true,
   "dependencies": {
-    "rails-ujs": "<%= Rails::VERSION::STRING %>"<% unless options[:skip_turbolinks] %>,
+    "rails-ujs": "6.0.0-alpha"<% unless options[:skip_turbolinks] %>,
     "turbolinks": "5.1.1"<% end -%><% unless skip_active_storage? %>,
-    "activestorage": "<%= Rails::VERSION::STRING %>"<% end -%><% unless options[:skip_action_cable] %>,
-    "actioncable": "<%= Rails::VERSION::STRING %>"<% end -%>
-  }
+    "activestorage": "6.0.0-alpha"<% end -%><% unless options[:skip_action_cable] %>,
+    "actioncable": "6.0.0-alpha"<% end -%>
```

Comment:
Why hardcode these versus using the Rails::VERSION::STRING?

Reply:
`Rails::VERSION::STRING` is currently `6.0.0.alpha` so it wasn't matching up with the package names. 

If we can rename the package version in the `package.json` file to match the `Rails::VERSION::STRING` then we woudn't need to hardcode these - but I am not sure if the `-` is significant from a npm/yarn point of view.